content/about.njk

```diff
@@ -34,5 +41,60 @@ layout: layouts/base.njk
             challenges to emphasize how rewarding it can be to explore different avenues in the software and cybersecurity
             world.
         </p>
+        <details>
+            <summary>expand for full club charter</summary>
```

Comment:
I think it is better for us to link the StuyActivities page, we need the exposure there anyways, and putting it here seems redundant